### ISSUE TITLE
'NA' instead of blanks in countrycode_data.csv

### ISSUE THREAD

**Issue by cjyetman** [vincentarelbundock/countrycode#91]
since [commit c79098be623ace41a0256a41d75144a09f712a91](https://github.com/vincentarelbundock/countrycode/commit/c79098be623ace41a0256a41d75144a09f712a91), [data/countrycode_data.csv](https://github.com/vincentarelbundock/countrycode/blob/master/data/countrycode_data.csv) has 'NA' for blanks, which appears to have broken...
```r
countrycode('Namibia', 'country.name', 'iso2c')
[1] NA
Warning message:
In countrycode("Namibia", "country.name", "iso2c") :
  Some values were not matched: Namibia
```

**Comment by cjyetman** [comment]
fix in #101

**Comment by vincentarelbundock** [comment]
thanks!
